### ISSUE TITLE
能否在moodycamel::ConcurrentQueue中提供一个size()的功能

### ISSUE THREAD

**Issue by LOWRUI** [cameron314/concurrentqueue#277]

**Comment by cameron314** [comment]
> Can you provide a size() function in moodycamel::ConcurrentQueue?

No, though `size_approx()` should be fairly accurate most of the time (and completely accurate without contention).
